mms_msg/simulation/sro.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+
+def create_async_data(example, n_ch_node=4):
+    """
+    Desynchronizes the microphone channels of an observation.
+    Args:
+        example: Meeting example after scenario_map_fn
+        n_ch_node: Number of channels per microphone array (necessary for multi-array application)
+    Returns: example dict with desynchronized signals as additional key
+
+    """
+    from paderwasn.synchronization.simulation import sim_sro
+    sros = example['sro_trajectory']
+    stos = example['sto']
+    async_sigs = []
+    sigs = example['audio_data']['observation']
+    example['audio_data']['observation_'] = {'synchronous': sigs}
```

Comment:
is the name `'observation_'` confusing? Would it be easier to understand if we add `asynchronous_observation`? Since this has to be applied after the scenario_map_fn, it could also be fine to overwrite `observation` with this `observation_` dict, or to put the async version into `observation` and add `sync_observation`.

What about async for other signals, e.g., speech_source? Is that required?

Reply:
Asynchronous signals are only possible if we have the reverberated signals, so in our case, those would be image/early. Since they depend on the scenario, I would stick to only the observation, here. If asynchronous targets are necessary during training, the sampling probably needs to be adapted anyways.